lua/no-neck-pain/util/color.lua

```diff
@@ -30,31 +30,49 @@ function C.matchIntegrationToHexCode(colorCode)
     return colorCode
 end
 
--- creates an highlight group `NoNeckPain` with the given `colorCode` and assign it to the side buffer of the given `id`.
+-- creates an highlight group `NNPBuffers` with the given `color` and assign it to the side buffer of the given `id`.
```

Comment:
```suggestion
-- creates an highlight group `NNPBuffers` with the given `backgroundColor` and assign it to the side buffer of the given `id`.
```

so it's ready for the `textColor` feature